src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
+        super(color);
+        this.side = side;
+
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Square");
```

Comment:
Use only eng in messages/code: Try to make sure that all messages are in English to keep the code universal and consistent.